Cache.js

```diff
@@ -152,9 +152,12 @@ define([
 				// the result from the put should be dictated by the master store and be unaffected by the cachingStore,
 				// unless the master store doesn't implement put
 				return result || cachedPutResult;
-			}, function () {
+			}, function (exception) {
 				// Assuming a rejection of a promise invalidates the local cache
-				cachingStore.remove((directives && directives.id) || this.getIdentity(object));
+				cachingStore.remove((directives && directives.id) || cachingStore.getIdentity(object));
+				if(exception) {
+					throw exception;	
```

Comment:
Would it be possible for you to update the `Cache` unit test so it covers this?

Reply:
I have no prior experience with the-intern - but I did my best updating the cache-testcases